spec/integration/waf_configuration_spec.rb

```diff
@@ -0,0 +1,107 @@
+require "spec_helper"
+
+RSpec.describe "WAF configuration" do
+  let(:waf_config) do
+    JSON.parse(File.read("spec/fixtures/waf_rules.json"))
+  end
```

Comment:
So is it a config or rules?

Reply:
Renamed it to rules. But overall the term "config" also includes "rules" in libddwaf terminology.